Oqtane.Client/Themes/Controls/Theme/ControlPanel.razor

```diff
@@ -147,8 +147,7 @@
         {
             if (PageState.Page.IsPersonalizable && PageState.User != null && UserSecurity.IsAuthorized(PageState.User, RoleNames.Registered))
             {
-                PageState.EditMode = true;
```

Comment:
I'm not sure how you set the `EditMode` then check it in the line after?!!

Reply:
As I said in another PR I'm trying to test the functionality from within the code to improve the quiality

Reply:
I am not sure what you mean by "testing the functionality from within the code"? Does this mean that you are not actually testing the code at run-time, but instead using static code analyzers to identify potential issues?

Reply:
I'm not testing the code at the runtime, sometimes I identifying the bugs by observation or static code analyzers

Reply:
Ok it sounds like the answer to my earlier question is that you did not experience any functional problems with the application ie. this is not an "issue" according to the standard definition. This is an optimization to code which is already working properly. In general my philosophy in these cases is "if it aint broke then dont fix it" as there are often unexpected side effects. In this case the change is very straightforward and easy to test/validate so I will merge the PR.